book/managers_guide.md

```diff
@@ -0,0 +1,119 @@
+
+Automating a process using code is not sufficient to achieve all of the benefits, without also applying adequate quality assurance.
+
+## Analytical code requires quality assurance
+
+We recommend that you ask your team the following questions to understand the quality assurance that is being applied in their analysis. Good quality assurance should be applied throughout an analysis project, therefore, it is important that the areas outlined below are reviewed regularly.
+
+The practices outlined below should be applied proportionately to the business risk and complexity of the project. Analytical code that doesn't apply quality assurance carries similar risk to other analysis appraoches. Your team should aim to outline what adequate quality assurance looks like early on in a project. These questions can then be used throughout the project to assess whether this target is being met, or whether the team are moving towards this goal.
+
+### Why have your chosen to do things this way?
+
+* The methodology and data should be suitable for the question being asked.
+* Your team should be using open-source analysis tools, with a reason for each given tool being chosen.
+* Users should be consulted throughout the development process, to ensure that their need is being met.
+
+### How is your code structured?
+
+* Logic should be written as functions, so that it can be reused consistently and tested.
+* Related functions should be grouped together in the same file, so that it is easy to find them.
+
+### How easy is it to adjust the way that the analysis runs?
+
+* Parts of the code that may change should be stored in separate configuration files, so that they can be changed without changing the code.
+* Things that often change in analysis code include input and output file paths, reference dates and model parameters.
+
+### How are you storing input data?
+
+* Input data should be versioned, so that analysis outputs can be reproduced.
+* Data should be stored in an open format (e.g. CSV or ODS), not formats that are restricted to proprietary software like SAS and Stata. 
+* Large or complex datasets should be stored in a database.
+
+### What does the documentation look like?
+
+* Every function should be documented in the code, so that it is clear what the function is supposed to do.
+* Function documentation should also include what goes in and what comes out of each function.
+* Where code will be run or re-used by others, documentation should include usage examples and test data.
+
+### How has peer review been done?
+
+* Technical colleagues should conduct internal peer reviews of the code.
+* Peer review of individual changes to the code will help to identify issues sooner. This is also more manageable than reviewing the complete final analysis.
+* If the product is high risk, an external peer review should also be conducted.
+* Peer review should follow a standard procedure, so that reviews are consistent.
```

Comment:
Perhaps add:
There should be evidence that peer reviews are acted on. When issues or concerns are raised, they should be addressed.